docroot/modules/custom/foia_webform/src/AgencyLookupService.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\foia_webform;
+
+use Drupal\node\Entity\Node;
+use Drupal\node\NodeInterface;
+
+/**
+ * Class AgencyLookupService.
+ */
+class AgencyLookupService implements AgencyLookupServiceInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getComponentFromWebform($webformId) {
+    $entityQueryService = \Drupal::service('entity.query');
```

Comment:
Since we're using a service here, we should pass it in as an argument in the `services.yml` file like we do elsewhere.